Exiled.API/Interfaces/IPlugin.cs

```diff
@@ -68,6 +68,11 @@ public interface IPlugin<out TConfig> : IComparable<IPlugin<IConfig>>
         /// </summary>
         TConfig Config { get; }
 
+        /// <summary>
+        /// Gets the plugin translations.
+        /// </summary>
+        ITranslations Translations { get; }
```

Comment:
By doing this, to get a translation string, you'd be forced to cast `Translations` to your `Translation` class.
Like: `((Translation)Translations).TranslationString`, which is not so intuitive or comfortable to do.

Reply:
I can't think of too many ways to do this so that translations are made optional. Maybe we can provide another plugin class that inherits from the main one, with a second generic type used in an override of Translations. I'm not too sure if that'll work though. I'll have to think a bit on how to solve this.

Reply:
Alright, I've made a basic implementation of what I described above. I'll need to do some testing on it later, but it should be working. Let me know if you see any issues with it. I'll continue going through the list of requested changes now as well.

Reply:
It's fine, if it works, I'll approve it.